### ISSUE TITLE
Pre-selecting which group to add a tab to

### ISSUE THREAD

**Issue by doso77** [josh-berry/tab-stash#152]
I'm not sure if this is already a feature and I just can't figure it out. I'm looking to select what group to stash a tab without having to drag and drop that specific tab to the group. 

### Problem Statement

It gets quite tedious to drag and drop a tab to a specific group when I have over 50+ group. Please someone correct me if there is an easier way to do this. I will select all the tabs I want to stash using crtl and select then I right click and select stash tabs. Now by default, these will all stash into a new group. Now, let's say I want to add all of these tabs to an existing group that is way down on my groups list. I have to individually drag and drop them to the group.

### Preferred Solution(s)

Perhaps having a drop down menu where you could quickly scroll to the group you can stash to. It would be great if this was also possible to do right through the toolbar.

### Additional Context

Again I apologize if this is already a feature and I am just oblivious! Thanks!:)

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
No, this is not a feature today, though I agree it would be nice to send tabs directly to the right place. I'll keep it in mind for a future release!

**Comment by lucatrv** [comment]
I agree with this feature request. The best solution IMHO would be to open a popup when stashing tabs, very similar to the one that Firefox opens when bookmarking a tab. This popup would allow to create a new group or select an existing one, proposing by default the last selected option. It would also allow to accept or cancel the operation, in order to prevent unwanted stashing.

**Comment by cronco** [comment]
I would really love some functionality around this. I am happy to help workshop some UX/implementation ideas if the maintainers would like some help with this.

**Comment by blitmap** [comment]
Would love this, I have a few too many groups and dragging a tab slowly to the bottom of the list to the correct group can get tedious.  Love this extension overall

**Comment by josh-berry** [comment]
So this isn't the same as having something in Firefox's context menu, but until this is implemented, you can stash directly to the right place using the Tab Stash UI itself (no need for drag-and-drop). This is even easier if you use [the experimental popup view](https://github.com/josh-berry/tab-stash/issues/115), which you can enable in the advanced settings.

In the Tab Stash UI, select each of the tabs you want to stash by clicking its icon. Then, find the stash you want and click its blue "Stash" icon. Or if you know exactly which stash you're looking for, you can open the selection menu (the blue ">>>" icon that appears next to the search bar when you have something selected) and search for the destination by name.

Hope this helps!

**Comment by blitmap** [comment]
> So this isn't the same as having something in Firefox's context menu, but until this is implemented, you can stash directly to the right place using the Tab Stash UI itself (no need for drag-and-drop). This is even easier if you use [the experimental popup view](https://github.com/josh-berry/tab-stash/issues/115), which you can enable in the advanced settings.
> 
> In the Tab Stash UI, select each of the tabs you want to stash by clicking its icon. Then, find the stash you want and click its blue "Stash" icon. Or if you know exactly which stash you're looking for, you can open the selection menu (the blue ">>>" icon that appears next to the search bar when you have something selected) and search for the destination by name.
> 
> Hope this helps!

Okay, this is awesome and does just what I need :)  Thank you!  I would say I never would have found this, but now that I know it's there I'm very happy.  I could suggest adding a button next to the tab group, so you can select every tab underneath it in "Open Tabs" or your particular groups.  Then you might see that you can select the tabs by their icon?  I'm happy though, you can probably close/resolve this issue :)
